subsys/mgmt/mcumgr/Kconfig

```
@@ -315,51 +315,67 @@ config MCUMGR_SMP_BT_AUTHEN
 	  Enables encrypted and authenticated connection requirement to
 	  Bluetooth SMP transport.
 
-config MCUMGR_SMP_BT_LATENCY_CONTROL
-	bool "Request low latency connection when handling SMP commands"
+config MCUMGR_SMP_BT_CONN_PARAM_CONTROL
+	bool "Request specific connection parameters for SMP packet exchange"
 	depends on SYSTEM_WORKQUEUE_PRIORITY < 0
+	depends on BT_GAP_PERIPHERAL_PREF_PARAMS
 	help
-	  Enables support for requesting low latency connection parameter when
+	  Enables support for requesting specific connection parameters when
 	  SMP commands are handled. This option allows to speed up the command
 	  exchange process.
 	  Its recommended to enable this if SMP is used for DFU.
 
-config MCUMGR_SMP_BT_LATENCY_CONTROL_DEFAULT_LATENCY
-	int "Default value for connection latency"
-	depends on MCUMGR_SMP_BT_LATENCY_CONTROL
-	default BT_PERIPHERAL_PREF_LATENCY if BT_GAP_PERIPHERAL_PREF_PARAMS
-	default 99
+if MCUMGR_SMP_BT_CONN_PARAM_CONTROL
+config MCUMGR_SMP_BT_CONN_PARAM_CONTROL_MIN_INT
+	int "Minimum connection interval for SMP packet exchange"
+	default 6
+	range 6 65535
+	help
+	  Minimum connection interval in 1.25ms units used during the exchange of SMP packets.
+
+config MCUMGR_SMP_BT_CONN_PARAM_CONTROL_MAX_INT
+	int "Maximum connection interval for SMP packet exchange"
+	default 9
+	range 6 65535
+	help
+	  Maximum connection interval in 1.25ms units used during the exchange of SMP packets.
+
+config MCUMGR_SMP_BT_CONN_PARAM_CONTROL_LATENCY
+	int "Slave latency for SMP packet exchange"
+	default 0
 	range 0 499
 	help
-	  The value is a default connection latency that is used when restoring
-	  from low latency mode.
+	  Slave latency used during the exchange of SMP packets.
+
+config MCUMGR_SMP_BT_CONN_PARAM_CONTROL_TIMEOUT
+	int "Supervision timeout for SMP packet exchange"
+	default 42
+	range 10 65535
```

Range: 0x000A to 0x0C80

```suggestion
	range 10 3200
```